src/docfx/lib/UrlUtility.cs

```diff
@@ -110,6 +111,69 @@ public static DependencyType FragmentToDependencyType(string fragment)
             return fragment != null && fragment.Length > 1 ? DependencyType.Bookmark : DependencyType.Link;
         }
 
+        public static string GetRelativeUrl(string relativeToUrl, string url)
```

Comment:
Why not just directly use Path.GetRelativeTo after removing the beginning `/`?

Reply:
`Path.GetRelativePath` produces different result, I initially used it, see the diff in [this report](https://github.com/dotnet/docfx/pull/4588#issuecomment-501539552)